src/commands/scanner/run/dfa.ts

```diff
@@ -155,6 +162,12 @@ export default class Dfa extends ScannerRunCommand {
 		} else if (SFGE_IGNORE_PARSE_ERRORS in process.env && process.env.SFGE_IGNORE_PARSE_ERRORS.toLowerCase() === 'true') {
 			sfgeConfig.ignoreParseErrors = true;
 		}
+		// Check the status of the flag first, since the flag being true should trump the environment variable's value.
+		if (this.flags['rule-disable-warning-violation'] != null) {
+			sfgeConfig.ruleDisableWarningViolation = this.flags['rule-disable-warning-violation'] as boolean;
+		} else if (SFGE_RULE_DISABLE_WARNING_VIOLATION in process.env && process.env.SFGE_RULE_DISABLE_WARNING_VIOLATION.toLowerCase() === 'true') {
+			sfgeConfig.ruleDisableWarningViolation = true;
+		}
```

Comment:
The logic looks identical to `ignoreParseErrors` handling. Can we extract the logic to a method, which can be invoked in both places? Will help avoid code duplication.

Reply:
Good idea.